packages/sources/react/package.json

```diff
@@ -17,8 +17,12 @@
     "type": "git",
     "url": "git+https://github.com/Decathlon/vitamin-web.git"
   },
-  "license": "Apache-2.0",
+  "sideEffects": [
+    "dist/packages/**/*.js",
```

Comment:
why do our own components are considered side effects here?

Reply:
Also, i'm fine with telling the consumer to import the vars, but it means a breaking change, depends if that's okay or not